src/content/docs/en/welcome.mdx

```diff
@@ -19,11 +19,11 @@ To do this, we stick to a set of values:
 
 ### Ultramarine is Pragmatic.
 
-Where Fedora tries to adhere to ideology (that’s perfectly fine), by shipping an (almost) entirely free system, Ultramarine works off the model of least resistance. That means we try to make it easy for the user, even if we have to ship some proprietary drivers and codecs that Fedora wouldn’t be comfy with out-of-box.
+While Fedora ships a primarily free system by default, Ultramarine works off the model of least resistance. We try to make it easy for the user, even if we have to ship some proprietary drivers, firmware, and codecs that Fedora wouldn’t be comfy with out-of-box.

```

Comment:
ideology is an important part of this, the addition of firmware is good but keep the rest the way it is